tendrl/gluster_integration/objects/gluster_brick/atoms/create/__init__.py

```diff
@@ -0,0 +1,105 @@
+from tendrl.commons.event import Event
+from tendrl.commons.message import Message
+from tendrl.commons import objects
+from tendrl.gluster_integration.objects.gluster_brick import GlusterBrick
+
+
+class Create(objects.BaseAtom):
+    obj = GlusterBrick
+    def __init__(self, *args, **kwargs):
+        super(Create, self).__init__(*args, **kwargs)
+
+    def run(self):
+        bricks = self.parameters.get('GlusterBrick.brick_dict')
+        brick_dict = {}
+        for k, v in bricks.iteritems():
+            host = k.split(":")[0]
+            if not brick_dict.has_key(host):
+                brick_dict[host] = {}
+            disk = v["disk"]
+            brick_path = k.split(":")[-1]
+            mount_path = "/" + "/".join(brick_path.split("/")[1:-1])
+            brick_dict[host].update({
+                disk: {
+                    "node_id": v["node_id"],
+                    "mount_path": mount_path,
+                    "brick_path": brick_path,
+                    "lv": "tendrl" + brick_path.replace("/", "_") + "_lv",
```

Comment:
Sure. Have raised an issue: https://github.com/Tendrl/gluster-integration/issues/221
Will take this up soon